docs/proposals/groups.md

```diff
@@ -0,0 +1,94 @@
+	// that allows all Users to inspect and modify OrgGroups of Organizations that they are an Admin for.
+	// Admins may NOT modify Organization membership.
+	AdminUsers util.StringSet
+	// AdminClusterGroups is a list of ClusterGroups that can administer this Organization.  By default a policy rule exists
+	// that allows all Users to inspect and modify OrgGroups of Organizations that they are an Admin for.
+	// Admins may NOT modify Organization membership.
+	AdminClusterGroups util.StringSet
+
+	// Members is a list of Users that are members of the Organization.  Only members may be assigned
+	// to OrgGroups.
+	Members util.StringSet
+}
+```
+
+The Organization has a list of admins.  A default policy rule is bound to all-authenticated users that allows them inspect and modify OrgGroups of Organizations that they are an Admin for (whether via User or ClusterGroup).  Having the policy rule described and bound in this way makes it easier to manage mutiple organizations without having a proliferation of roles and rolebindings.
+
+The Organization has a list members.  Only users from this list may be assigned to OrgGroups.  This prevents an overzelous organization admin from creating OrgGroups made up on non-members.  By default, an organization admin does not have rights to change Organization membership.
+
+*****TBD*****: How are users added to the Organization???  Being able to modify the OrgGroup structure of known members seem logically distinct from being able to force users into an Organization.  I wouldn't expect the same person to do both.
+
+Organizations will have an `organizations/groups` subresource.  A `GET` returns the list of OrgGroups.  A `POST` with a OrgGroup creates a new OrgGroup.  A `PUT` with a OrgGroup updates a new OrgGroup.  Subresources can't be named, so we do not have an individual get for `organizations/groups`.  This approach will require enhancements to `kubectl`, but prevents a Organization admins from being able to perform a `list` operation against `orggroups`.  The alternative is to have `list` on `orggroups` only show the `OrgGroups` you're allowed to see, but that would be different from all other `list`s.
+
+
+## Tying Projects to Organizations
+A Project may have at most one owner (Organization).  An Organization may own many projects.  We will indicate project ownership by having an "openshift.io/owner" label that contains a value of the form "resource/namespace/name".  Initially, the only valid resource will an Organization, so the value will look like: "organizations//MyOrg".  This allows for easy listing of projects owned by a particular Organization and allows us to extend the concept of project ownership to other resource types at a later time.
+
+
+## Using OrgGroups in RoleBindings
+RoleBindings will get a new field:
+```
+	// OrgGroups holds all the OrgGroups directly bound to the role.
+	OrgGroups util.StringSet
```

Comment:
> Why we need that additional field? We already have Groups there, and since we've stated we are going to prefix all OrgGroups with Organization name, can't we use the Groups fields for that?

Having a separate field means that we will not have to prefix the names.   Not prefixing the names makes it possible to export from one project and create in another project without getting rejected by admission for invalid group references.